collections/javascript-game-engines/index.md

```diff
@@ -22,6 +22,7 @@ items:
  - Cloud9c/taro
  - replit/kaboom
  - straker/kontra
+ - excaliburjs/excaliburjs.github.io
```

Comment:
This is the repo that generates the website , not the code itself. You should add: 
```suggestion
 - excaliburjs/Excalibur
```